tsfc/fem.py

```diff
@@ -328,6 +329,11 @@ def callback(key):
     return iterate_shape(mt, callback)
 
 
+@translate.register(Zero)
+def _translate_zero(terminal, mt, params):
```

Comment:
I will make this a bit more general, so that not just `Zero`s, but all constant values are handled.